csv_datasets/ldbc/lists/edges-sf-003-full-without-types.txt

```diff
@@ -75,28 +75,28 @@ creationDate|:START_ID(Person)|:END_ID(Post)|:TYPE
 sf-003/dynamic/person_likes_post/part-00000-42cce85f-f27a-40d3-8c65-69f139ffd456-c000.csv.split_0
 sf-003/dynamic/person_likes_post/part-00000-42cce85f-f27a-40d3-8c65-69f139ffd456-c000.csv.split_1
 
-creationDate|:START_ID(Person)|:END_ID(Language)|:TYPE
+:IGNORE|:START_ID(Person)|:END_ID(Language)|:TYPE
```

Comment:
where is this coming from?

Reply:
Second Milos. I don't see a "Language" node in the spec.

Reply:
@gliga @roshandathathri  this looks like one of those edges that don't exist in the actual schema but came out of the csv generator for some reason (idk why the generator would generate something not in the schema, @tylershunt / @danielmawhirter might have more context)